event/producer_test.go

```diff
@@ -63,13 +62,12 @@ func TestProducer_SearchDataImport(t *testing.T) {
 		}
 		Convey("When SearchDataImport is called on the event producer", func() {
 
-			err := searchDataImportProducer.SearchDataImport(ctx, searchDataImportTestEvent)
+			err := searchDataImportProducer.SearchDataImport(ctx, expectedSearchDataImportEvent)
 			So(err, ShouldBeNil)
 
 			var avroBytes []byte
 			select {
 			case avroBytes = <-pChannels.Output:
-				fmt.Printf("avroBytes: %v\n", avroBytes)
 				log.Event(ctx, "avro byte sent to producer output", log.INFO)
 			case <-time.After(time.Second * 5):
 				log.Event(ctx, "failing test due to timed out", log.INFO)
```

Comment:
I know you didn't add this log line in this PR but would you mind improving it and basically combining it with t.FailNow in line below to `t.Fatalf("failing test due to timing out after %v seconds", timeoutAfter)` where `var testTimeout = 5 *time.Second` - can reuse variable in select statement like so: `case <-time.After(testTimeout):`

Reply:
General rule when writing tests use t.log to logout messages and remember you can always combine a log with forcing the test fail.

Reply:
Thanks. done the changes